Example-Energy-REXX/README.md

```diff
@@ -0,0 +1,123 @@
+## Example-Energy-REXX
+
+This samples queries a CPC and lists the energy management attributes for that CPC. It stores the results, which are in .csv format, in a z/OS data set member.  You can import the content into Microsoft™ Excel or another application of your choice.
+
+This samples uses **HWIREST API** to do the following:
+- List CPCs and retrieve the URI and target name associated with the LOCAL CPC or the *CPCname* provided
+- For the designated CPC, retrieve the following information:
+     - cpc-power-rating
+     - cpc-power-consumption
+     - cpc-power-saving
+     - cpc-power-saving-state
+     - cpc-power-save-allowed
+     - cpc-power-capping-state
+     - cpc-power-cap-minimum
+     - cpc-power-cap-maximum
+     - cpc-power-cap-current
+     - cpc-power-cap-allowed
+     - zcpc-power-rating
+     - zcpc-power-consumption
+     - zcpc-power-saving
+     - zcpc-power-saving-state
+     - zcpc-power-save-allowed
+     - zcpc-power-capping-state
+     - zcpc-power-cap-minimum
+     - zcpc-power-cap-maximum
+     - zcpc-power-cap-current
+     - zcpc-power-cap-allowed
+     - zcpc-ambient-temperature
+     - zcpc-exhaust-temperature
+     - zcpc-humidity
+     - zcpc-dew-point
+     - zcpc-heat-load
+     - zcpc-heat-load-forced-air
+     - zcpc-heat-load-water
+     - zcpc-maximum-potential-power
+     - zcpc-maximum-potential-heat-load
+     - last-energy-advice-time
+     - zcpc-minimum-inlet-air-temperature
+     - zcpc-maximum-inlet-air-temperature
+     - zcpc-maximum-inlet-liquid-temperature
+     - zcpc-environmental-class   
+
+
+## System Prep work
+- Store RXENRGY1 into a data set
+- Ensure your z/OS user ID has at least READ access to the following FACILITY Class Profiles
+    - HWI.TARGET.netid.nau
+    - HWI.TARGET.netid.nau.imagename
+
+    <p>where netid.nau represents the 3– to 17– character SNA name of the particular CPC who's LPARs are being audited and imagename represents the 1– to 8- character LPAR name of all the LPARs available on that CPC </p>
```

Comment:
given you're not access any image information, you don't need anything related to the LPAR facility class profile